axon-framework/messaging-concepts/supported-parameters-annotated-handlers.md

```diff
@@ -37,9 +37,13 @@ By default, `@CommandHandler` annotated methods allow the following parameter ty
 
   For more specifics on this it is recommend to read [this](message-intercepting.md#commandhandlerinterceptor-annotation) section.
 
-* If the application is run in a Spring environment, any Spring Bean can be resolved.
+* If the application is run in a Spring environment, any Spring Bean can be resolved. 
+  
+> If Spring beans are resolved for message handling methods `@Autowired` annotation should be used to get the revamped solution. 
+> 
+> The `@Qualifier` annotation can be used in conjunction with this to further specify which Bean should be resolved.
 
-  Note that the `@Qualifier` annotation can be used in conjunction with this to further specify which Bean should be resolved.
+* The newly added `ScopeDescriptor` can only be implemented for Aggregates and Sagas at this time.
```

Comment:
I'd change this a little bit. For now, it is new, but in a year or two the `ScopeDescriptor` is no longer new at all.
So, what about something like this:

```suggestion
* A parameter of type `ScopeDescriptor` can be resolved. This can be used when scheduling a deadline through the `DeadlineManager`. Note that the `ScopeDescriptor` only makes sense from within the scope of an Aggregate or Saga.
```

Maybe even add a link to the deadlines page in the second sentence there...what do you think?